Basic/ClientDriven/Assets/Scripts/ServerIngredientSpawner.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using Unity.Netcode.Samples;
+using UnityEngine;
+using Random = System.Random;
+
+public class ServerIngredientSpawner : ClientServerBaseNetworkBehaviour
+{
+    [SerializeField]
+    private List<GameObject> m_SpawnPoints;
+
+    [SerializeField]
+    private float m_SpawnRatePerSecond;
+
+    [SerializeField]
+    private GameObject m_IngredientPrefab;
+
+    private float m_LastSpawnTime;
+    private Random r = new Random();
+
+    private void FixedUpdate()
+    {
+        if (NetworkManager != null && !IsServer) return;
```

Comment:
Why the !IsServer technically it should not be enable since you are a `ClientServerBaseNetworkBehavior`

Reply:
updated